Makefile

```diff
@@ -1,5 +1,5 @@
 all:
-	gcc -Wall -O3 -o usbtool usbtool.c -ludev
+	$(CC) -Wall -O3 -o usbtool usbtool.c -ludev
```

Comment:
When updating Makefile perhaps refactor it to a more proper format. For example https://github.com/tenox7/ttyplot/blob/master/Makefile

Reply:
Yes. You're right.